requirements.txt

```diff
@@ -1,4 +1,4 @@
-jupyter==1.0.0
+jupyterlab==3.0.1
```

Comment:
As @triplepoint is using 3.0.3 in the webapp-side adjustments, should we use it here too?